packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -229,6 +229,14 @@ module.exports = {
                         ],
                       },
                     },
+                    {
+                      loader: require.resolve('sass-loader'),
+                      options: {
+                        includePaths: [
+                          path.resolve(__dirname, '../src/app/scss'),
```

Comment:
@mcallegari10 
Is this loading only from this path? or imported .sass in component will work too?

Reply:
This is for the scss absolute imports.
```scss
@import "colors.scss"; // with this config
@import "../../../scss/colors.scss"; // without it
```